signer/migrations/0003__create_tables.sql

```diff
@@ -128,12 +133,14 @@ CREATE TABLE sbtc_signer.rotate_keys_transactions (
     FOREIGN KEY (txid) REFERENCES sbtc_signer.transactions(txid) ON DELETE CASCADE
 );
 
+-- TODO: Unused?
 CREATE TABLE sbtc_signer.deposit_responses (
     response_txid BYTEA NOT NULL,
     deposit_txid BYTEA NOT NULL,
     deposit_output_index INTEGER NOT NULL
 );
 
+-- TODO: Unused?
 CREATE TABLE sbtc_signer.withdrawal_responses (
```

Comment:
We've basically renamed these `swept_deposits` and `swept_withdrawals`, so we can remove them.

Reply:
Yeah I didn't even notice these until later :sweat_smile: removed in [9340bab](https://github.com/stacks-network/sbtc/pull/710/commits/9340bab1fb8245358c026b5dd3d1e68f91258a20)